specification/appendix/commitment_discounts.md

```diff
@@ -0,0 +1,274 @@
+# Commitment Discounts
+
+A commitment discount is a billing discount that offers reduced rates on preselected SKUs in exchange for an obligated usage or spend amount over a predefined term. Commitment discounts typically consist of a set of purchase and usage records within cost and usage datasets.
```

Comment:
Would love this definition to not include the word `discount` in the first line. Something like:

```suggestion
A commitment discount is a rate reduction on preselected SKUs in exchange for an obligated usage or spend amount over a predefined term. Commitment discounts typically consist of a set of purchase and usage records within cost and usage datasets.
```

Reply:
I'm going to keep this as is to maintain consistency with this pr, the new `CommitmentDiscunt*` columns, and the FinOps terminology page, so actually need to add the missing word, "model".

<img width="657" alt="image" src="https://github.com/user-attachments/assets/63c8fc92-7551-43ee-b730-4e12c0128fd3">